scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -19,7 +20,7 @@ import scala.meta.{Importer, Mod, Pkg, Term, Tree}
 /**
   * Produces formatted output from sequence of splits.
   */
-class FormatWriter(formatOps: FormatOps) {
+class FormatWriter(formatOps: FormatOps)(implicit style: ScalafmtConfig) {
```

Comment:
you don't need this. `formatOps` contains the configuration, via `initStyle`, and it's even imported directly two lines below.

Reply:
wow, thank you